js/structured-filter.js

```diff
@@ -153,7 +155,7 @@ $.widget( 'evol.structFilter', {
 				that._field=that._getFieldById(fieldID);
 				var fType=that._type=that._field.type;
 				that._setEditorOperator();
-				if(fType==fTypes.list || fType==fTypes.bool){
+				if([fTypes.list, fTypes.bool, fTypes.exList, fTypes.selList].indexOf(fType) > -1){
```

Comment:
Alternatively could be (here and 177):
````javascript
if(fType==fTypes.list || fType==fTypes.bool || fType==fTypes.exList || fType==fTypes.selList){}
````
though I think array is more succinct and easier to maintain?